CONTRIBUTING.md

```diff
@@ -582,3 +582,6 @@ Keep in mind that better variable names can reduce the need for comments, e.g.:
 that are not licensed under compatible terms will be rejected.  Moreover,
 contributions will not be accepted unless _all_ authors accept the project's
 contributor license agreement.
+
+# Use of AI-code Generation
+The Stacks Foundation has a very strict policy of not accepting AI-generated code PRs due to uncertainly about licensing issues.
```

Comment:
Should this be a subheading under `Licensing`?

```suggestion

## Use of AI-code Generation
The Stacks Foundation has a very strict policy of not accepting AI-generated code PRs due to uncertainly about licensing issues.
```

Reply:
@kantai Sure! I just did this quickly. Please edit as needed.